Tone/core/context/ToneAudioNode.test.ts

```diff
@@ -320,6 +330,15 @@ describe("ToneAudioNode", () => {
 				disconnect(gain, output);
 			});
 		});
+
+		it("can fan in multiple nodes to a destination", async () => {
+			const output = new Gain();
+			const input0 = new Gain();
+			const input1 = new Gain();
+			const input2 = new Gain();
+			fanIn(output, input0, input1, input2);
+			expect(output.numberOfInputs).to.equal(3);
```

Comment:
`numberOfInputs` refers to the number of input channels, not connections, so i wouldn't expect the number of inputs to equal 3, it should still equal 1 like it did before.